### DISCUSSION TITLE
Caught plotting error: bad allocation

### DISCUSSION THREAD

**Discussion by Siom-lab** [Chia-Network/chia-blockchain#5493] [Support]
I have an issue. 
Windows 10 has 64 GB of ram. 30 GB RAM of plotter

Starting plotting progress into temporary dirs: D:\Chia and D:\Chia
ID: 8126c8c7747c32d348c574e71cf45c466d73c7e7835a7fb69d78854352ca4e12
Plot size is: 32
Buffer size is: 30600MiB
Using 128 buckets
Using 6 threads of stripe size 65536

Starting phase 1/4: Forward Propagation into tmp files... Tue May 18 13:18:23 2021
Computing table 1
F1 complete, time: 97.958 seconds. CPU (194.45%) Tue May 18 13:20:00 2021
Computing table 2
Caught plotting error: bad allocation
[7764] Failed to execute script chia
Traceback (most recent call last):
  File "chia\cmds\chia.py", line 81, in <module>
  File "chia\cmds\chia.py", line 77, in main
  File "click\core.py", line 829, in __call__
  File "click\core.py", line 782, in main
  File "click\core.py", line 1259, in invoke
  File "click\core.py", line 1259, in invoke
  File "click\core.py", line 1066, in invoke
  File "click\core.py", line 610, in invoke
  File "click\decorators.py", line 21, in new_func
  File "chia\cmds\plots.py", line 135, in create_cmd
  File "chia\plotting\create_plots.py", line 176, in create_plots
RuntimeError: bad allocation

If anone know how to resolve that issue ? I will be grateful

**Comment by number435398** [comment]
I just had the same error happen to me.  I was already doing about 5 plots and my additional plots that were starting up started failing almost immediately.  I had 64gb of ram and only about half of it was being used. My temp drive had plenty of space (over 2tb), and my secondary drive was virtually empty with nearly 2tb on it, and my final destination had several TB free. Oh and my processor wasn't even maxed yet.

**Comment by BrandtH22** [comment]
@Siom-lab , if you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
